.travis.yml

```diff
@@ -14,6 +14,10 @@ services:
 
 jobs:
   include:
+    - stage: PythonLint
+      os: linux
+      script: "[[ -z \"$(git ls-files | xargs egrep -l '\s+$')\" ]]"
```

Comment:
Still not too sure about this, I tried running this without the whitespace changes to see if it failed, but I guess it saw there were no changes and didn't run Travis.

Reply:
I think travis might have failed to parse the yaml

Reply:
This run is from an older version: https://travis-ci.org/cocotb/cocotb/jobs/632467451#L183